backend/src/main/java/heartbeat/controller/pipeline/PipelineController.java

```diff
@@ -27,9 +29,9 @@ public class PipelineController {
 
 	private final BuildKiteService buildKiteService;
 
-	@GetMapping("/{pipelineType}")
+	@PostMapping("/{pipelineType}")
```

Comment:
Please update `String` to a customized type, like `BoardType`

and update the frontend code, pass the `buildkite` as the parameter, not `buildKite`
thx